packages/namespaces/src/preventNamespace.js

```diff
@@ -0,0 +1,13 @@
+import { mergeDeepLeft } from 'ramda';
+
+import { NAMESPACE_PREVENTED } from './constants';
+
+/**
+ * Associates an action with a default namespace, overwriting any previous namespace.
+ *
+ * @param {Object} action action to add the default namespace to
+ */
+const preventNamespace = action =>
+	mergeDeepLeft({ meta: { namespace: NAMESPACE_PREVENTED } }, action);
```

Comment:
please use `mergeNamespace` here so we can work with functions too. just like in attachNamespace

Reply:
done